.env.development

```diff
@@ -0,0 +1,22 @@
+REACT_APP_ELASTIC_APM_SERVICE_VERSION=${ELASTIC_APM_SERVICE_VERSION}
```

Comment:
Allow using Elastic Stack common environment variables to pass to the React application in development mode.